spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/firestore/it/FirestoreDocumentationIntegrationTests.java

```diff
@@ -43,6 +43,7 @@
  */
 public class FirestoreDocumentationIntegrationTests {
 	private final ApplicationContextRunner contextRunner = new ApplicationContextRunner()
+			.withPropertyValues("spring.cloud.gcp.firestore.project-id=spring-cloud-gcp-ci-firestore")
```

Comment:
We tried before to externalize CI-sepcific config. See: https://github.com/spring-cloud/spring-cloud-gcp/blob/master/.travis.yml#L42.
Can we make that work?

Reply:
Yes, added a new profile in `autoconfig` module.